.travis.yml

```diff
@@ -69,7 +69,7 @@ script:
   - mkdir build && pushd build &&
     cmake -DCMAKE_PREFIX_PATH="$HOME/kokkos;$HOME/scafacos"
           -DCabana_ENABLE_Serial=OFF ${CMAKE_OPTS[@]}
-          -DCabana_ENABLE_MPI=ON
+          -DCabana_ENABLE_MPI=ON -DCabana_ENABLE_FORTRAN_EXAMPLES=ON
```

Comment:
Same need to be added to Jenkins CI.

Reply:
@dalg24

Reply:
@junghans  How to do the Jenkins CI?

Reply:
@dalg24 please tell us what to do!

Reply:
@junghans @dalg24 Do we have to do something about this, or we can leave it for later?

Reply:
@dal24 @junghans Looks like it is already in the Jenkins CI now.